src/java/htsjdk/samtools/cram/structure/AlignmentSpan.java

```diff
@@ -0,0 +1,92 @@
+package htsjdk.samtools.cram.structure;
+
+import htsjdk.samtools.SAMRecord;
+
+/**
+ * A span of reads on a single reference.
+ */
+public class AlignmentSpan {
```

Comment:
the class is used in more than one package, for example:
htsjdk.samtools.cram.encoding.reader.RefSeqIdReader
htsjdk.samtools.CRAMIndexer

Not sure how to refactor this without 'public' modifier. I guess htsjdk.samtools.util.Interval could be used instead but it lacks record count.